askama_derive/src/generator/filter.rs

```diff
@@ -627,6 +626,70 @@ impl<'a> Generator<'a, '_> {
         );
         Ok(DisplayWrap::Unwrapped)
     }
+
+    fn visit_default_filter(
+        &mut self,
+        ctx: &Context<'_>,
+        buf: &mut Buffer,
+        args: &[WithSpan<'a, Expr<'a>>],
+        node: Span<'_>,
+    ) -> Result<DisplayWrap, CompileError> {
+        const ARGUMENTS: &[&FilterArgument; 3] = &[
+            FILTER_SOURCE,
+            &FilterArgument {
+                name: "default_value",
+                default_value: None,
+            },
+            &FilterArgument {
+                name: "boolean",
```

Comment:
Why not naming it "do_unwrap" or something along the line to better match what it's doing instead?

Reply:
Question still open. :)

Reply:
The filter only makes sense if it behaves the same as in jinja.

Reply:
That saddens me, but it's a valid point.